portal-ui/src/screens/Console/Account/ChangeUserPasswordModal.tsx

```diff
@@ -0,0 +1,170 @@
+  closeModal: () => void;
+  setModalErrorSnackMessage: typeof setModalErrorSnackMessage;
+}
+
+const ChangePassword = ({
+  classes,
+  open,
+  //selectedUser,
+  closeModal,
+  setModalErrorSnackMessage,
+}: IChangePasswordProps) => {
+  const [newPassword, setNewPassword] = useState<string>("");
+  const [reNewPassword, setReNewPassword] = useState<string>("");
+  const [loading, setLoading] = useState<boolean>(false);
+
+  const changePassword = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (loading) {
+      return;
+    }
+    setLoading(true);
+
+    let request: ChangeUserPasswordRequest = {
+      //selected_user: selectedUser,
+      new_secret_key: newPassword,
+    };
+
+    api
+      .invoke("POST", "/api/v1/account/change-user-password", request)
+      .then((res) => {
+        setLoading(false);
+        // setSelectedUser("");
+        setNewPassword("");
+        setReNewPassword("");
+        closeModal();
+      })
+      .catch((err) => {
+        setLoading(false);
+        setNewPassword("");
+        setReNewPassword("");
+        setModalErrorSnackMessage(err);
+      });
+  };
+
+  return open ? (
+    <ModalWrapper
+      title="Change Password"
+      modalOpen={open}
+      onClose={() => {
+        setNewPassword("");
+        setReNewPassword("");
+        closeModal();
+      }}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description"
+    >
+      <form
+        noValidate
+        autoComplete="off"
+        onSubmit={(e: React.FormEvent<HTMLFormElement>) => {
+          changePassword(e);
+        }}
+      >
+        <Grid container>
+          <Grid item xs={12} className={classes.formScrollable}>
+              <Grid item xs={12}>
+              <InputBoxWrapper
+                id="new-password"
+                name="new-password"
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  setNewPassword(event.target.value);
+                }}
+                label="New Password"
+                type="password"
+                value={newPassword}
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <InputBoxWrapper
+                id="re-new-password"
+                name="re-new-password"
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  setReNewPassword(event.target.value);
+                }}
+                label="Type New Password Again"
+                type="password"
+                value={reNewPassword}
+              />
+            </Grid>
+          </Grid>
+          <Grid item xs={12} className={classes.buttonContainer}>
+            <Button
+              type="submit"
+              variant="contained"
+              color="primary"
+              disabled={
+                loading ||
+                !(
+                  newPassword.length > 0 &&
+                  reNewPassword.length > 0
```

Comment:
we can add another condition to disable the button if passwords don't match
```suggestion
                  reNewPassword.length > 0 && newPassword !== reNewPassword
```